openslides_backend/action/user/delete_temporary.py

```diff
@@ -0,0 +1,14 @@
+from ...models.user import User
+from ..action import register_action
+from ..default_schema import DefaultSchema
+from ..generics import DeleteAction
+
+
+@register_action("user.delete_temporary")
+class UserDeleteTemporary(DeleteAction):
```

Comment:
@FinnStutzenstein Do we really need this action? It is likely the same as UserDelete?